internal/rbd/replication.go

```diff
@@ -102,32 +99,30 @@ func DisableVolumeReplication(rbdVol *rbdVolume,
 		// disabled the image on all the remote (secondary) clusters will get
 		// auto-deleted. This helps in garbage collecting the volume
 		// replication Kubernetes artifacts after failback operation.
-		localStatus, rErr := rbdVol.GetLocalState()
+		localStatus, rErr := rv.GetLocalState()
 		if rErr != nil {
-			return nil, status.Error(codes.Internal, rErr.Error())
+			return fmt.Errorf("%w: %w", ErrLocalStateNotFound, rErr)
```

Comment:
```suggestion
			return fmt.Errorf("%w: failed to ... : %w", ErrLocalStateNotFound, rErr)
```
To maintain consistency and better tracing.

Reply:
Thanks for the comment Rakshith, just to clarify : We already have this definition in errors.go -> failed to get local state
Did you meant to put that(failed to.. statement) twice here, as it will be retrieved from ErrLocalStateNotFound too. 
@Rakshith-R

Reply:
In that case of repeating" failed to ..."
It's fine.

and @riya-singhal31 failed to get is not equivalent to not found.

Please change the error name to something like `ErrFetching..` instead of `Err..NotFound`.
NotFound in the error definition is misleading IMO.

Reply:
Updated, thanks. PTAL